ui/shared/status/StatusLoadingIndicator.qml

```diff
@@ -1,14 +1,13 @@
 import QtQuick 2.13
 import "."
 
-SVGImage {
-    id: loadingImg
-    source: "../app/img/loading.svg"
-    width: 17
+StatusIcon {
+    id: root
+    icon: "loading"
     height: 17
-    fillMode: Image.Stretch
+    width: 17
```

Comment:
Maybe we can use `implicitWidth`? Why i mention it:
`StatusLoadingIndicator {
         width: someSituation ? 15 : implicitWidth
         ....`
 What do you think? its make sense?

Reply:
Unfortunately, that doesn't work when component's `implicitWidth` and `implicitHeight` properties are read-only, which is the case of this component: https://github.com/status-im/status-desktop/pull/2435#discussion_r620994414